### ISSUE TITLE
the trait `IntoSchemaManagerConnection<'_>` is not implemented for `&sea_orm::DatabaseConnection`

### ISSUE THREAD

**Issue by Shaphil** [koddlers/rust-actix#1]
I am getting the following error while trying to run an Actix-Web application after integrating Sea-ORM,

```bash
error[E0277]: the trait bound `&sea_orm::DatabaseConnection: IntoSchemaManagerConnection<'_>` is not satisfied
   --> src/main.rs:21:18
    |
21  |     Migrator::up(&db, None).await.unwrap();
    |     ------------ ^^^ the trait `IntoSchemaManagerConnection<'_>` is not implemented for `&sea_orm::DatabaseConnection`
    |     |
    |     required by a bound introduced by this call
    |
    = help: the following other types implement trait `IntoSchemaManagerConnection<'c>`:
              &'c migration::sea_orm::DatabaseConnection
              &'c migration::sea_orm::DatabaseTransaction
              SchemaManagerConnection<'c>
note: required by a bound in `migration::MigratorTrait::up`
   --> C:\Users\Shaphil\.cargo\registry\src\index.crates.io-6f17d22bba15001f\sea-orm-migration-0.12.15\src\migrator.rs:230:12
    |
228 |     async fn up<'c, C>(db: C, steps: Option<u32>) -> Result<(), DbErr>
    |              -- required by a bound in this associated function
229 |     where
230 |         C: IntoSchemaManagerConnection<'c>,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `MigratorTrait::up`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `rust-actix` (bin "rust-actix") due to 1 previous error
```

This is the line that produces the error,

```rust
Migrator::up(&db, None).await.unwrap();
```

The application runs fine if I comment out that line.

Here is the `main.rs` file - [https://github.com/koddlers/rust-actix/blob/master/src/main.rs](https://github.com/koddlers/rust-actix/blob/master/src/main.rs)

I was following along a tutorial on Youtube, [https://www.youtube.com/watch?v=ENgFBKTBqDU&list=PLGOIZXklfFkRh8jHNY8070KUl86Tj3Ztf](https://www.youtube.com/watch?v=ENgFBKTBqDU&list=PLGOIZXklfFkRh8jHNY8070KUl86Tj3Ztf)

What am I doing wrong?

**Comment by Shaphil** [comment]
Solved by downgrading to `v0.12.15`.

Changed from this,

```toml
sea-orm = { version = "1.0.0", features = ["sqlx-postgres", "runtime-tokio-rustls", "macros"] }
```

to this,

```toml
sea-orm = { version = "0.12.15", features = ["sqlx-postgres", "runtime-tokio-rustls", "macros"] }
```

**Comment by Shaphil** [comment]
Solved by downgrading to `v0.12.15`.

Changed from this,

```toml
sea-orm = { version = "1.0.0", features = ["sqlx-postgres", "runtime-tokio-rustls", "macros"] }
```

to this,

```toml
sea-orm = { version = "0.12.15", features = ["sqlx-postgres", "runtime-tokio-rustls", "macros"] }
```
